src/visit_vtk/lightweight/vtkVisItCellLocator.C

```diff
@@ -1456,18 +1441,18 @@ void vtkVisItCellLocator::GetOverlappingBuckets(double x[3], int vtkNotUsed(ijk)
   numberOfBucketsPerPlane = this->NumberOfDivisions*this->NumberOfDivisions;
   leafStart = this->NumberOfOctants
     - numberOfBucketsPerPlane*this->NumberOfDivisions;
-  
+
   // Initialize
   this->Buckets->Reset();
-  
+
   // Determine the range of indices in each direction
   for (i=0; i < 3; i++)
     {
-    minLevel[i] = (int) ((double) (((x[i]-dist) - this->Bounds[2*i])
+    minLevel[i] = static_cast<int>((((x[i]-dist) - this->Bounds[2*i])
```

Comment:
It looks like original intent may have been to perform the arithmetic inside the parenthesis in double precision. But, the original code cast to double only outside the parenthesis and so your adjustment is consistent with that. I just wonder if maybe we should consider if the original intent was missed and it would really be something like...

```suggestion
    minLevel[i] = static_cast<int>(((((double) x[i]-dist) - this->Bounds[2*i])
```

Reply:
static_cast the double

Reply:
The cast to double was useless, as everything being used inside is already double.